tempodb/encoding/vparquet/combiner.go

```diff
@@ -99,6 +99,21 @@ func (c *Combiner) ConsumeWithFinal(tr *Trace, final bool) (spanCount int) {
 		return
 	}
 
+	// coalese root level information
+	if tr.EndTimeUnixNano > c.result.EndTimeUnixNano || c.result.EndTimeUnixNano == 0 {
```

Comment:
```suggestion
	if tr.EndTimeUnixNano > c.result.EndTimeUnixNano {
```

I think this covers it.